.github/workflows/pr.yml

```
@@ -1,23 +1,31 @@
 
 jobs:
   build:
     runs-on: windows-latest
 
+    strategy:
+      matrix:
+        arch: [x64, ARM64]
+
     steps:
       - name: Checkout
         uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)